### PR TITLE
Re-adding missing process fields

### PR DESCRIPTION
**Pull request by kgeller**
Resolves https://github.com/elastic/ecs/issues/1902

Adding two process fields to the top-level so that their details show in the documentation

See updated docs https://ecs_1906.docs-preview.app.elstc.co/guide/en/ecs/master/ecs-process.html

### CODE REVIEW

File: experimental/generated/ecs/ecs_flat.yml [hunk 1]
```diff
@@ -9725,6 +9741,36 @@ process.real_user.name:
...
+    This field exists to the benefit of EQL and other rule engines since it''s not
+    possible to compare equality between two fields in a single document. e.g `process.entity_id`
+    = `process.group_leader.entity_id` (top level process is the process group leader)
+    OR `process.entity_id` = `process.entry_leader.entity_id` (top level process is
+    the entry session leader)
+
+    Instead these rules could be written like: `process.group_leader.same_as_process:
+    true` OR `process.entry_leader.same_as_process: true`
+
+    Note: This field is only set on `process.entry_leader`, `process.session_leader`
```
Comment (andrewkroh) [inline experimental/generated/ecs/ecs_flat.yml:9764]: I think we should omit this from the generated schemas for `process`.

Comment (ebeahan) [reply experimental/generated/ecs/ecs_flat.yml:9764]: I'm exploring how to describe a field in the ECS docs but not include it in the generated schema.

I created https://github.com/elastic/ecs/issues/1907 to track.

**Comment by ebeahan** [comment]
Updated PR to use the `docs_only` attribute.

**Comment by ebeahan** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.3|[](https://github.com/elastic/ecs/pull/1929)|
|✅|8.2|[](https://github.com/elastic/ecs/pull/1930)|


### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**